guides/v3.4.0/components/passing-properties-to-a-component.md

```diff
@@ -54,6 +54,21 @@ known as being "bound"). That is, if the value of `componentProperty`
 changes in the component, `outerProperty` will be updated to reflect that
 change. The reverse is true as well.
 
+In addition to making properties available, actions can be made available
+to components. This allows data to flow back up to its parent. You pass actions
+like this.
+
+```handlebars {data-filename=app/templates/index.hbs}
+  {{button-with-confirmation
+    text="Click here to unsubscribe."
+    onConfirm=(action "unsubscribe")
+  }}
+```
+
+It is important to note that actions can only be passed from a controller or another
+component. They cannot be passed from a route. See [passing an action to the component](/triggering-changes-with-actions/#toc_passing-the-action-to-the-component)
```

Comment:
I just had another look locally and this already looks great! One more thing I noticed is, that the link here doesn't point to the correct page, yet:

![linktoactions](https://user-images.githubusercontent.com/8811742/46414855-509cbf00-c724-11e8-8b1c-fe3644e95ec6.gif)

Can you check what is missing for the link? I assume there are other relative links in the guides source which you can use as a reference, too.